packages/create/src/generators/packageJson.test.ts

```diff
@@ -0,0 +1,95 @@
+import { describe, it } from 'vitest';
+import { createFixture } from 'fs-fixture';
+import { generatePackageJson } from './packageJson.js';
+
+describe('generatePackageJson', () => {
+  it('should generate package.json for webpack template', async ({
+    expect,
+  }) => {
+    const fixture = await createFixture({});
+
+    await generatePackageJson(fixture.path, {
+      projectName: 'my-app',
+      template: 'webpack',
+    });
+
+    const packageJsonContent = await fixture.readFile('package.json', 'utf8');
+
+    expect(packageJsonContent).toMatchInlineSnapshot(`
+      "{
+        "name": "my-app",
+        "version": "1.0.0",
+        "private": true,
+        "scripts": {
+          "start": "sku start",
+          "build": "sku build",
+          "test": "sku test",
+          "format": "sku format",
+          "lint": "sku lint"
+        }
+      }
+      "
+    `);
+
+    await fixture.rm();
```

Comment:
If you use [explicit resource management](https://github.com/privatenumber/fs-fixture?tab=readme-ov-file#using-keyword-explicit-resource-management) then this cleanup call can be removed.